CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# dbt_hubspot v0.12.0
```

Comment:
Also, this should be 0.13.0 for this release

Reply:
updated